client/src/index.html

```diff
@@ -59,10 +59,12 @@
 
         <div class="flex-footer" ng-cloak>
           <ul>
-            <li><a class="session" ng-click="AppCtrl.settings()">
-              <span class="glyphicon glyphicon-cog"></span>
-              <span class="link-label">User Settings</span>
-            </a></li>
+            <li>
+              <a class="session" ng-click="AppCtrl.settings()">
+                <span class="glyphicon glyphicon-cog"></span>
+                <span class="link-label">User Settings</span>
```

Comment:
In this case, there is already a `SETTINGS.TITLE` key that can be substituted in.  I think this has been plain text since the initial app structure PR.  I've changed it locally and will push as soon as your review is done.

It seems to me all developers should at least translate a single key as either `fr.json` or `en.json`, so that we know what they were trying to say.  No need to do both - we can clean the entire application's translations before "publishing" a new version of bhima.